Sources/TSCBasic/WritableByteStream.swift

```diff
@@ -790,7 +790,11 @@ public final class LocalFileOutputByteStream: FileOutputByteStream {
     override final func writeImpl(_ bytes: ArraySlice<UInt8>) {
         bytes.withUnsafeBytes { bytesPtr in
             while true {
+              #if os(Android)
+                let n = fwrite(bytesPtr.baseAddress!, 1, bytesPtr.count, filePointer)
```

Comment:
This force-unwrap works fine on linux too, but I separated it out anyway in case there is concern it breaks other platforms.

Reply:
I see this done elsewhere in this repo so should be fine, doing the force unwrap on all platforms now.

Reply:
`ArraySlice` currently never passes a buffer with a nil `baseAddress`. It's arguably an implementation detail, but we rely on it in many places.

Reply:
OK, I only modified this because the compiler complained and errored. Is there something you'd like me to change here?

Reply:
No, I just wanted to further clarify why the force unwrap is correct here.